csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -4212,6 +4212,43 @@ The relevant path for this test is:
 
 > A tool MAY add all products of the affected group of this vulnerability to the remediation as quick fix.
 
+#### 6.1.30 Integer and Semantic Versioning
+
+It must be tested that all elements of type `/$defs/version_t` follow the same versioning system. Integer versioning and semantic versioning SHALL NOT occur within the same document.
```

Comment:
@sthagen Should we use `SHALL NOT` or `MUST NOT`in the second sentence? The original requirement from section [3.1.11](https://docs.oasis-open.org/csaf/csaf/v2.0/cs01/csaf-v2.0-cs01.html#3111-version-type) states:

> A CSAF document MUST use only one versioning system.

Reply:
Here we may profit from using a single sentence like so:

It must be tested that all elements of type `/$defs/version_t` follow either integer versioning or semantic versioning homogeneously within the same document.